### PR TITLE
exercises: Assert precision to two decimals

### LINKED ISSUE #562: space-age: Compare values using explicit precision

### PR DESCRIPTION
**Pull request by ErikSchierboom**
While mentoring some space-age solutions, I found that it sometimes failed due to a missing precision specifier. This PR adds a precision check and sets the existing ones to two decimals.

Closes #562.

### CODE REVIEW

File: generators/Templates/_AssertEqualWithin.liquid [hunk 1]
```diff
@@ -1 +1 @@
-{{ Sut }} |> should (equalWithin 0.000000001) {{ Expected }}
\ No newline at end of file
+{{ Sut }} |> should (equalWithin 0.01) {{ Expected }}
```
Comment (robkeim) [inline generators/Templates/_AssertEqualWithin.liquid]: What's the reason for this change? Just to not see as many 0s?

Comment (ErikSchierboom) [reply generators/Templates/_AssertEqualWithin.liquid]: Correct!

**Comment by robkeim** [comment]
Nice, thanks @ErikSchierboom! 🎊

**Comment by Fernando-Soto** [comment]
Did you bump up the version number? My old one was 1.1.0 and so was this one?

**Comment by ErikSchierboom** [comment]
That is correct, as the version number is tied to the version of the canonical data.

**Comment by Fernando-Soto** [comment]
So the only way to make sure that nothing in the test suite has changed from the last time I used it while mentoring in F# is to reload the test suite and student code?

**Comment by ErikSchierboom** [comment]
Correct.
